backend/WebApplication1/Infrastructure/Loggers/Abstract/ILogger.cs

```diff
@@ -0,0 +1,9 @@
+﻿using static WebApplication1.Models.Log;
+
+namespace WebApplication1.Infrastructure.Loggers.Abstract
+{
+    interface ILogger
+    {
+        void Log(LoggableEntity identifiedEntity, int identifiedEntityID);
```

Comment:
if it was generic, you could do void Log(T entity) and pass it altogether with person id